ballerina/http_connection.bal

```diff
@@ -184,6 +185,18 @@ public isolated client class Caller {
     }
 }
 
+isolated function retrieveMediaTypeForApplicationError(string? annotatedMediaType, anydata retrievedMediaType) returns string? {
```

Comment:
Ins't retrievedMediaType type should be `string?` ?

Reply:
Fixed with [28f2606e58a20d925a1569c3070c9fe670884718](https://github.com/ballerina-platform/module-ballerina-http/pull/839/commits/28f2606e58a20d925a1569c3070c9fe670884718)